.github/workflows/java_binding_publish.yml

```diff
@@ -128,4 +136,117 @@ jobs:
           ./gradlew publishToSonatype
           if [ "$RUNNER_OS" == "macOS" ];
           then ./gradlew publishToSonatype -PbuildTarget=aarch64-apple-darwin
-          fi
\ No newline at end of file
+          fi
+
+  release-android:
+    name: Create Android releases
+    needs: release-desktop
```

Comment:
Is this just for `needs.release-desktop.outputs.tag`?

Reply:
yes